src/message/buttons/urlButton.js

```diff
@@ -35,7 +35,13 @@ export default class UrlButton extends Button {
         return this;
     }
 
+    setMessengerExtensions(value = false) {
```

Comment:
since it's not a required field, just `enableMessengerExtensions` should be fine